### PR TITLE
Add CI and fix linter/staticheck issues

### PR DESCRIPTION
**Pull request by kdisneur**

### CODE REVIEW

**Comment by kdisneur** [comment]
@Kemosabert I added some `??` in the documentation of the `monitoring` package because I didn't know what these structs and functions were used for. 

If you can suggest some documentation I would appreciate any help :)

File: platform/application.go [hunk 1]
```diff
@@ -21,18 +21,21 @@ import (
 	"github.com/fewlinesco/go-pkg/platform/web"
 )
 
+// ApplicationConfig represents a minimal API configuration that can be override / augmented by the aplication
```
Comment (arnaudmorisset) [inline platform/application.go:24]:
```suggestion
// ApplicationConfig represents a minimal API configuration that can be override / augmented by the application
```

File: platform/application.go [hunk 2]
```diff
@@ -21,18 +21,21 @@ import (
...
 type ApplicationConfig struct {
 	API             web.ServerConfig  `json:"api"`
 	Monitoring      web.ServerConfig  `json:"monitoring"`
 	Tracing         tracing.Config    `json:"tracing"`
 	ErrorMonitoring monitoring.Config `json:"error_monitoring"`
 }
 
+// ClassicalApplicationConfig represents a classical API configuration including a SQL Database and a broker that can be override / augmented by the aplication
```
Comment (arnaudmorisset) [inline platform/application.go:32]:
```suggestion
// ClassicalApplicationConfig represents a classical API configuration including a SQL Database and a broker that can be override / augmented by the application
```

**Comment by kdisneur** [comment]
Thanks @arnaudmorisset and sorry for the typo.

**Pull request merged**